Controller/Payment/Order.php

```diff
@@ -27,14 +27,12 @@ class Order extends \Razorpay\Magento\Controller\BaseController
      * @param \Magento\Framework\App\CacheInterface $cache
      * @param \Magento\Sales\Api\OrderRepositoryInterface $orderRepository
      * @param \Psr\Log\LoggerInterface $logger
-     * @param \Magento\Catalog\Model\Session $catalogSession
      */
     public function __construct(
         \Magento\Framework\App\Action\Context $context,
         \Magento\Customer\Model\Session $customerSession,
         \Magento\Checkout\Model\Session $checkoutSession,
         \Razorpay\Magento\Model\Config $config,
-        \Magento\Catalog\Model\Session $catalogSession,
```

Comment:
@ramth05 Why it is moved from catalog session to checkout session?

Reply:
@ChetanGN , As we are dealing with checkout (quote) module for payments , we need to use same session. Catalog session used for products filtering etc.